routing/control_tower.go

```diff
@@ -57,6 +57,10 @@ type ControlTower interface {
 	// sent out immediately.
 	SubscribePayment(paymentHash lntypes.Hash) (*ControlTowerSubscriber,
 		error)
+
```

Comment:
General comment about commit message: typically they are prefixed with the subsystem(s).

Reply:
Fixed